main/main.go

```diff
@@ -158,15 +157,10 @@ func createFrontend() *tbfe {
 	w, h := termbox.Size()
 	t.handleResize(h, w, true)
 
-	t.console.Buffer().AddObserver(&t)
+	t.console.AddObserver(&t)
 	t.setupCallbacks(t.currentView)
 
-	path := path.Join(backend.LIME_PACKAGES_PATH, "themes", "TextMate-Themes", "Monokai.tmTheme")
-	if sc, err := textmate.LoadTheme(path); err != nil {
-		log.Error(err)
-	} else {
-		scheme = sc
-	}
+	t.editor.Settings().Set("color_scheme", path.Join("..", "packages", "themes", "TextMate-Themes", "Monokai.tmTheme"))
```

Comment:
We have set color scheme in [default settings](https://github.com/limetext/lime-packages/blob/master/Default/Preferences.sublime-settings#L27) so no need to set it manually here.